src/utils/itemFilterUtils.ts

```diff
@@ -15,71 +19,92 @@ export const combineRecapLocations = (locations: string[]) => {
   recapLocations: string
 ) => {
   const locs = location.map((loc) => {
     if (loc === "Offsite") return recapLocations
     else return loc
   })
-  const location_query = locs.length ? "item_location=" + locs.join(",") : ""
-  const format_query = format.length ? "item_format=" + format.join(",") : ""
-  const status_query = status.length ? "item_status=" + status.join(",") : ""
 
-  const query = [location_query, format_query, status_query]
-    .filter((q) => q)
-    .join("&")
-  if (query.length) return encodeURI("?" + query)
-  else return ""
+  return {
+    ...(locs.length && { item_location: locs.join(",") }),
+    ...(format.length && { item_format: format.join(",") }),
+    ...(status.length && { item_status: status.join(",") }),
+    ...(year.length && { item_date: year.join(",") }),
+  }
 }
 
 // numItems default is for development purposes only. Once data is being
 // passed in to the Item Filters components, this default should be removed.
-export const buildItemsMatchedStringString = (query, numItems = 20) => {
-  const items = `Item${numItems === 1 ? "" : "s"}`
-  if (Object.keys(query).length === 0) return `${numItems} ${items}`
-  const num = numItems === 0 ? "No" : numItems
+export const buildItemsMatchedStringString = (
+  query: ItemFilterQueryParams,
+  numItemsMatched = 0
+) => {
+  const items = `Item${numItemsMatched === 1 ? "" : "s"}`
+  if (Object.keys(query).length === 0) return `${numItemsMatched} ${items}`
+  const num = numItemsMatched === 0 ? "No" : numItemsMatched
   return `${num} Matching ${items} `
 }
 
-export const buildAppliedFiltersString = (
+export const buildAppliedFiltersTagSetData = (
   appliedFilters: CollapsedMultiValueAppliedFilters,
-  itemAggs: ItemFilterData[]
-) => {
-  const filters = Object.keys(appliedFilters)
-    .map((field: string) => {
-      const appliedFilterPerField = appliedFilters[field]
-      if (appliedFilterPerField.length) {
-        const fieldAggregations = itemAggs.find(
-          (agg: ItemFilterData) => agg.field === field
-        )
-        const labels = fieldAggregations.labelsForConcatenatedValues(
-          appliedFilterPerField
-        )
-        return field + ": " + labels
+  itemAggregations: ItemFilterData[]
+): TagSetFilterDataProps[] => {
+  const filters: TagSetFilterDataProps[] = []
+  Object.keys(appliedFilters).forEach((field: string) => {
+    const appliedFilterPerField = appliedFilters[field]
+    appliedFilterPerField.forEach((filterValue: string) => {
+      const fieldAggregations = itemAggregations.find(
+        (aggregation: ItemFilterData) => aggregation.field === field
+      )
+      const valueLabel =
+        field === "year"
+          ? filterValue
+          : fieldAggregations?.labelForValue(filterValue)
+      if (valueLabel) {
+        filters.push({
+          label: `${capitalize(field)} > ${valueLabel}`,
+          id: kebabCase(`${field}-${valueLabel}`),
+          iconName: "close",
```

Comment:
From the docs, you don't need this property and the console also warns you, so make sure to check the messages:
<img width="642" alt="Screenshot 2024-07-10 at 6 31 16 PM" src="https://github.com/NYPL/research-catalog/assets/1280564/5796446a-a4fa-4ec6-a023-99e401b3ef3f">